service-registry/src/main/resources/META-INF/maven/io/servicecomb/java-chassis/pom.properties

```diff
@@ -0,0 +1,3 @@
+groupId=io.servicecomb
```

Comment:
why we need this file ?

Reply:
in mavenVersionFor's source code，it need to read pom.properties,
public static Version mavenVersionFor(ClassLoader classLoader, String groupId, String artifactId) {
    InputStream pomPoperties = classLoader.getResourceAsStream("META-INF/maven/" + 
        groupId.replaceAll("\\.", "/") + "/" + artifactId + "/pom.properties");

Reply:
i remember this file will be generated automatically.

Reply:
yes,when mvn package,this file will be generated automatically, i'll delete it